examples/js/exporters/GLTFExporter.js

```diff
@@ -1028,6 +1004,74 @@ THREE.GLTFExporter.prototype = {
 
 			}
 
+			var forceIndices = options.forceIndices;
+			var isMultiMaterial = Array.isArray( mesh.material );
+
+			if ( ! forceIndices && isMultiMaterial ) {
+
+				// temporal workaround.
+				console.warn( 'THREE.GLTFExporter: Force index for a mesh with multi-material.', mesh );
```

Comment:
Let's change this message to:

> THREE.GLTFExporter: Creating index for non-indexed multi-material mesh.

("Force" sounds like a we are telling the user to do something, rather than telling them what we are doing)

And, right now this message will appear if the mesh is already indexed.. let's hide the message in that case.

Reply:
Oops, I've missed adding `geometry.index === null`.

Reply:
Updated, thanks.